.github/workflows/housekeeping.yml

```
@@ -1,7 +1,12 @@
 name: Housekeeping
 
 on:
-  pull_request:
+  # pull_request_target is used to allow forks write permissions when running
+  # this workflow. With the pull_request trigger, forks do not have any write
+  # access for security reasons, however write access is needed in order to
+  # request reviews. Since this workflow is simply requesting reviewers, it is
+  # safe to allow forks write access. 
```

This may fix yamllint

```suggestion
  # safe to allow forks write access.
```

---

that's what I get for doing it quickly in vim haha